lib/api/controllers/session_controller.ex

```diff
@@ -1,12 +1,29 @@
 defmodule NabooAPI.SessionController do
   use Phoenix.Controller
+  use PhoenixSwagger
 
   alias Naboo.Accounts
   alias NabooAPI.Auth.Sessions
   alias NabooAPI.Views.Errors
 
   require Logger
 
+  swagger_path(:sign_in) do
+    post("/login")
+    summary("Log in")
+    description("Log in with an account known in the database")
+    produces("application/json")
+    deprecated(false)
+    parameter(:email, :body, :string, "email of the account", required: true)
+    parameter(:password, :body, :string, "password of the account", required: true)
+
+    response(200, "token.json", %{},
+      example: %{
+        token: "46853121354qsfqse"
```

Comment:
Please use a JWT token here, as it makes more sense for the reader, like this:
`eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c`